### PR TITLE
chore: remove unused variables in react-liveness

### PR DESCRIPTION
**Pull request by thaddmt**
#### Description of changes
- Fixed various codeql lint errors for unused variables - https://github.com/aws-amplify/amplify-ui/security/code-scanning

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [ ] Tests are updated
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/react-liveness/src/components/FaceLivenessDetector/service/utils/__tests__/liveness.test.ts [hunk 1]
```diff
@@ -20,15 +19,15 @@ describe('Liveness Helper', () => {
     });
   });
 
-  describe('getFaceMatchStateInLivenessOval', () => {
```
Comment (0618) [inline packages/react-liveness/src/components/FaceLivenessDetector/service/utils/__tests__/liveness.test.ts:23]: nit: if we plan to uncomment it in the future, shall we add a `TODO` to say when?

**Review by 0618 — DISMISSED** [review]

File: examples/next/pages/ui/components/liveness/components/LivenessDefault.tsx [hunk 2]
```diff
@@ -20,16 +18,12 @@ export default function LivenessDefault({
     stopLiveness,
   } = useLiveness();
 
-  const [error, setError] = useState(undefined);
-  const [checkFailed, setCheckFailed] = useState(false);
```
Comment (reesscot) [inline examples/next/pages/ui/components/liveness/components/LivenessDefault.tsx:24]: I'm guessing this was never used anywhere?

Comment (thaddmt) [reply examples/next/pages/ui/components/liveness/components/LivenessDefault.tsx:24]: It was previously, but when we removed it we forgot to remove the useState

File: packages/react-liveness/src/components/FaceLivenessDetector/service/utils/__tests__/liveness.test.ts [hunk 3]
```diff
@@ -20,15 +19,15 @@ describe('Liveness Helper', () => {
...
-    const ovalDetails: LivenessOvalDetails = {
-      height: 254,
-      width: 157,
-      centerY: 285, // top: 285 - 254 / 2 = 158
-      flippedCenterX: 286, // left: 286 - 157 / 2 = 207.5
-      centerX: 286, // left: 286 - 157 / 2 = 207.5
-    };
-  });
+  // describe('getFaceMatchStateInLivenessOval', () => {
```
Comment (thaddmt) [inline packages/react-liveness/src/components/FaceLivenessDetector/service/utils/__tests__/liveness.test.ts:22]:
```suggestion
  // TODO: add test for this
  // describe('getFaceMatchStateInLivenessOval', () => {
```

**Review by reesscot — APPROVED** [review]

File: examples/next/pages/ui/components/liveness/components/LivenessDefault.tsx [hunk 4]
```diff
@@ -64,12 +58,9 @@ export default function LivenessDefault({
                   const response = await handleGetLivenessDetection(
                     createLivenessSessionApiData.sessionId
                   );
-                  if (!response.isLive) {
-                    setCheckFailed(true);
-                  }
                 }}
                 onError={(error) => {
-                  setError(error);
+                  console.error(error);
```
Comment (ioanabrooks) [inline examples/next/pages/ui/components/liveness/components/LivenessDefault.tsx:63]: Are we not surfacing errors to end users?

Comment (thaddmt) [reply examples/next/pages/ui/components/liveness/components/LivenessDefault.tsx:63]: We have a default error modal that shows up, this is just another error callback for developers who want custom error handling

**Review by ioanabrooks — APPROVED** [review]

**Review by hbuchel — APPROVED** [review]

**Pull request merged**